lib/src/base-styles/_nx-tables.scss

```diff
@@ -183,6 +189,11 @@
   -ms-user-select: none;
 }
 
+.nx-cell--filter-header {
```

Comment:
Or, if you rearranged the classes a bit, you could get rid of that `nth-child` stuff.  What if instead of this modifier class on the cells themselves you had a modifier class on the row? So `nx-table-row--filter-header`.  Then these styles here could be on the selector `.nx-table-row--filter-header > .nx-cell--header`.  The changes to the sticky styles above could be undone and then an additional block like this could be added:

```
.nx-table-row--filter-header > .nx-cell--header {
  top: 45px;
}
```

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/a954210b17845f9ed60e40fd1bf15de1f5723c9a